src/Generate.jl

```diff
@@ -67,17 +67,17 @@ function update!(eco::Ecosystem, timestep::Unitful.Time)
             # Check if grid cell currently active
             if eco.abenv.active[x, y] && (eco.cache.totalE[i, 1] > 0)
                 # Calculate effective rates
-                birthprob = params.birth[j] * timestep * adjusted_birth
-                deathprob = params.death[j] * timestep * adjusted_death
+                birthrate = params.birth[j] * timestep * adjusted_birth
+                deathparam = params.death[j] * timestep * adjusted_death
 
-                # Put probabilities into 0 - 1
-                newbirthprob = 1.0 - exp(-birthprob)
-                newdeathprob = 1.0 - exp(-deathprob)
+                # Turn deathparam into probability and cancel units of birthrate
+                birthrate += 0.0
```

Comment:
I don't understand what this is for... which obviously makes me nervous that it's a mistake!

Reply:
It's just to make sure the units cancel out into a Float64 - sometimes it doesn't do this automatically, unless there's some sort of sum involved.

Reply:
That is weird!